src/shims/manual/Directory.Build.props

```diff
@@ -29,6 +29,10 @@
     <DefineConstants Condition="'$(TargetsNetCoreApp)' == 'true'">$(DefineConstants);netcoreapp</DefineConstants>
   </PropertyGroup>
 
+  <ItemGroup>
+    <Reference Include="netstandard" />
```

Comment:
I see why you did this.  I think an alternate fix would be to just modify the include statement below to include netstandard.dll in ReferencePath.  Either way has the same effect.